pkgs/development/python-modules/netaddr/default.nix

```
@@ -4,33 +4,22 @@
 , pytest
 , fetchpatch
 , glibcLocales
+, pythonOlder
+, importlib-resources
```

Remove unused fetchpatch
I tend to sort the arguments into something like:

```
, builders
, pythonPackages
, checkInputs
```

```suggestion
, pythonOlder
, glibcLocales
, importlib-resources
, pytestCheckHook
```